src/utils/retrieveCredentialsInfo.js

```diff
@@ -0,0 +1,67 @@
+import i18next from 'i18next';
+
+// Compares array of IDs with data in localstorage
+export const _matchCredentialId = (app, id, setData, cleanupTime, CredentialsAPI) => {
```

Comment:
It may be a misunderstanding, but this is the only const name which shouldnt be start with underscore